base/db/R/get.trait.data.pft.R

```diff
@@ -136,15 +136,26 @@ get.trait.data.pft <- function(pft, modeltype, dbfiles, dbcon, trait.names,
         } else {
           # Check if PFT membership has changed
           PEcAn.logger::logger.debug("Checking if PFT membership has changed.")
-          existing_membership <- utils::read.csv(
-            need_paths[["pft_membership"]],
-            # Columns are: id, genus, species, scientificname
-            # Need this so NA values are formatted consistently
-            colClasses = c("double", "character", "character", "character"),
-            stringsAsFactors = FALSE,
-            na.strings = c("", "NA")
-          )
-          diff_membership <- symmetric_setdiff(
+          if (pfttype == "plant") {
```

Comment:
There seems to be a lot more duplicate code here than necessary. Could we just have the `if` be used to define the colClasses variable, and then use that variable in the `existing_membership` load?

Reply:
Okay, fixed to be much more compact. Thanks!